### PR TITLE
Guardian UI Wizard shell

### PR DESCRIPTION
**Pull request by garyray-k**
Building on the Guardian UI initialization. This aims to lay a basic skeleton for the wizard itself.

This also resolved a Prettier and Eslint conflict I was encountering.

To examine:

1. Navigate to `guardian-ui`
2. `npm run start` (this should launch a browser to localhost:3000`
3. Click buttons

### CODE REVIEW

**Comment by garyray-k** [comment]
This uses a common pattern in React with `useContext` and `useReducer` to manage state. There are other solutions for state management, but given the minimal state required for this app, I think vanilla React should work fine.

File: guardian-ui/src/App.tsx [hunk 1]
```diff
@@ -1,22 +1,145 @@
...
+				<Box textAlign='center' fontSize='xl'>
+					<ColorModeSwitcher justifySelf='flex-end' />
+					<VStack spacing={8}>
+						<GuardianFlow />
+					</VStack>
+				</Box>
+			</GuardianProvider>
+		</ChakraProvider>
+	);
+};
```
Comment (justinmoon) [inline guardian-ui/src/App.tsx:31]: If we're going to use tabs, can we figure out how to configure GitHub to display them better? This is kinda hard to read with 8-space indentation.

Comment (garyray-k) [reply guardian-ui/src/App.tsx:31]: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference

You can update it in your personal settings.

**Comment by garyray-k** [comment]
Closing OBE.

**Pull request closed without merging**